src/uct/sm/mm/posix/mm_posix.c

```diff
@@ -33,11 +33,10 @@
                                                        open fd symlink from procfs */
 #define UCT_POSIX_SEG_FLAG_SHM_OPEN     UCS_BIT(62) /* use shm_open() rather than open() */
 #define UCT_POSIX_SEG_FLAG_HUGETLB      UCS_BIT(61) /* use MAP_HUGETLB */
-#define UCT_POSIX_SEG_FLAG_PID_NS       UCS_BIT(60) /* use PID NS in address */
-#define UCT_POSIX_SEG_FLAGS_MASK        (UCT_POSIX_SEG_FLAG_PROCFS | \
-                                         UCT_POSIX_SEG_FLAG_SHM_OPEN | \
-                                         UCT_POSIX_SEG_FLAG_PID_NS | \
-                                         UCT_POSIX_SEG_FLAG_HUGETLB)
+#define UCT_POSIX_SEG_FLAG_NS           UCS_BIT(60) /* use PID and USER NS in address */
+#define UCT_POSIX_SEG_FLAGS_MASK \
```

Comment:
why need to rename?

Reply:
renamed UCT_POSIX_SEG_FLAG_PID_NS to more generic UCT_POSIX_SEG_FLAG_NS as now pid and user namespace infos are sent together, will force push before further review.

Reply:
removed altogether